lib/buildkite/test_collector/test_links_plugin/formatter.rb

```diff
@@ -23,7 +23,9 @@ def dump_failures(notification)
       # return if suite url is nil
       return if metadata['suite_url'].nil?
 
-      @output << "\n\nTest Analytics failures:\n\n"
+      @output << "\n\n🔥 \x1b[31mTest Analytics failures:\n"
```

Comment:
hahahaha do we need MORE fire???

Reply:
What are you imagining?? Open to any suggestions!

Reply:
I personally like a 🔥 Test Analytics failures 🔥 
or a ⚠️ Test Analytics failures ⚠️

Reply:
I like the double flame!
<img width="580" alt="Screenshot 2023-11-08 at 9 28 00 AM" src="https://github.com/buildkite/test-collector-ruby/assets/25017977/e3405787-e0bc-45c1-8c76-6699e2ea5ecf">